recipe/meta.yaml

```diff
@@ -23,7 +24,7 @@ requirements:
     - setuptools_scm
 
   run:
-    - pathlib2  # [py27]
+    - pathlib2
```

Comment:
On only need `pathlib2` on `py27`:

https://github.com/ESSS/pytest-regressions/blob/3a89b58605dea6a4732d4fab90a19c6c93d0c9d0/src/pytest_regressions/common.py#L7-L10

Why did you remove the selector? 🤔

Reply:
To set as noarch package (would fix MacOS build failing and I'd to use this package on Win32). Since pytest-datadir is noarch, I thought that this one should be the same

But yeah. Seems weird to add a unneeded dependency to build a noarch. And pytest-datadir is probably broken on Python 2.7

Reply:
> would fix MacOS build failing and I'd like to use this package on Win32

Let's see if we can fix this by switching to Azure first: https://github.com/conda-forge/pytest-regressions-feedstock/pull/9

Reply:
Azure built successfully in #9 for all platforms, I think we can close this now. 👍

Reply:
No win-32 though :(